applications/FluidDynamicsApplication/custom_elements/fic.h

```diff
@@ -0,0 +1,451 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Jordi Cotela
+//                   Ignasi de Pouplana
+//
+
+#ifndef KRATOS_FIC_FLUID_ELEMENT_H
+#define KRATOS_FIC_FLUID_ELEMENT_H
+
+#include "includes/define.h"
+#include "includes/element.h"
+#include "includes/serializer.h"
+#include "geometries/geometry.h"
+
+#include "includes/cfd_variables.h"
+#include "custom_elements/fluid_element.h"
+#include "fluid_dynamics_application_variables.h"
+
+namespace Kratos
+{
+
+///@addtogroup FluidDynamicsApplication
+///@{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+template< class TElementData >
+class FIC : public FluidElement<TElementData>
```

Comment:
Could you add a minimal documentation, maybe just referring the reader to the corresponding paper? (A FIC-based stabilized finite element formulation for turbulent flows, 2017)